test/james.format.xml.tests.pas

```diff
@@ -53,19 +53,45 @@ procedure TXMLComponentTest.Document;
       + '  <Z>10.00</Z> '#13
       + '</Foo> ';
 var
-  Fmt: TFormatSettings absolute DefaultFormatSettings;
+  Fmt: TFormatSettings absolute
+    {$IFDEF FPC}
+      DefaultFormatSettings
+    {$ELSE}
+      FormatSettings
+    {$ENDIF};
 begin
   Fmt.DecimalSeparator := '.';
   with TXMLComponent.Create(XML) do
   try
-    with Document.FindNode('Foo') do
+    with Document.{$IFNDEF FPC}ChildNodes.{$ENDIF}FindNode('Foo') do
```

Comment:
The same I've said here https://github.com/mdbs99/james/pull/56#discussion_r127972920.